packages/application-shell/src/components/application-shell-authenticated/application-shell-authenticated.tsx

```diff
@@ -33,7 +33,7 @@ import ConfigureIntlProvider from '../configure-intl-provider';
 import ErrorApologizer from '../error-apologizer';
 import FetchProject from '../fetch-project';
 import FetchUser from '../fetch-user';
-import NavBar from '../navbar';
+import NavBar from '../navbar/navbar';
```

Comment:
I think we should import from "../navbar". Otherwise we're not taking advantage of lazy loading

Reply:
Oh that's a great catch! Thanks Kacper - [79571a2](https://github.com/commercetools/merchant-center-application-kit/pull/3277/commits/79571a2ad2a25a718d841e3dcde6259ed07d3a7b)